core/templates/local_vector.h

```diff
@@ -49,13 +49,9 @@ class LocalVector {
 	T *data = nullptr;
 
 public:
-	T *ptr() {
-		return data;
-	}
-
-	const T *ptr() const {
-		return data;
-	}
+	T *ptrw() { return data; }
```

Comment:
There should be some kind of note here on how this is a misnomer for the sake of uniform API